eos_data_distribution/SimpleStore/__init__.py

```diff
@@ -40,9 +41,9 @@ def __init__(self, base=None, prefix='/',
 
         if base: self.publish_all_names(base)
 
-    def _path_to_name(self, filename):
-        assert filename.startswith(self.base)
-        file_path = filename[len(self.base):].lstrip('/')
+    def _path_to_name(self, filename, basedir):
+        assert filename.startswith(basedir)
```

Comment:
maybe we should do realpath() here ? now that method has a subtle bug if called in another contexct than init -> publish_all_names()